### DISCUSSION TITLE
Relay with prisma

### DISCUSSION THREAD

**Discussion by aramis-it** [prisma/prisma#3613] [Q&A] [answered]
I am new to Prisma. I am thinking about the stack with react, relay, express, prisma, mysql. Where to start? Specially how to integrate relay with prisma?

**Comment by ryands17** [comment, marked as answer]
Hey @ulziibadrakh-p 👋 

Prisma isn't about GraphQL so it shouldn't impact the way you're using it and implementing your GraphQL API with Relay. You could use any stack like Express or Apollo and integrate Prisma with that. You could take any generic tutorial on a Relay implementation and replace the database access part with Prisma :)

Let me know if that works for you.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
